### ISSUE TITLE
List all tags and locations

### ISSUE THREAD

**Issue by vinayak-mehta** [vinayak-mehta/conrad#20]

**Comment by nishant-sethi** [comment]
can you please provide some description for this.

**Comment by vinayak-mehta** [comment]
@nishant-sethi Yes! There could be an extra command called `conrad list` which can be used to list all available tags and locations in the database. Locations can be formed by concatenating city, state and country.

Example usage:

<pre>
$ conrad list --tags
$ conrad list --locations
</pre>

Please feel free to suggest a better alternative.

**Comment by nishant-sethi** [comment]
@vinayak-mehta thanks, I'm picking this up.

**Comment by nishant-sethi** [comment]
@vinayak-mehta Please check my PR

**Comment by vinayak-mehta** [comment]
Thanks for the PR, I'll check it today! :)

**Comment by JamieWalker01** [comment]
Has the task for this issue changed?

**Comment by nik-ahuja** [comment]
Hi, @vinayak-mehta  I would like to pick this up if it's still pending.

**Comment by vinayak-mehta** [comment]
Hey everyone, sorry for the late reply. Nishant had raised a [PR](https://github.com/vinayak-mehta/conrad/pull/27) but it didn't do what the issue suggests (i.e. list down unique tags / locations in the db). I closed it back then because I didn't have time to get in a review cycle. 

At this point, I'm not sure if this is a useful feature. What do you all think? Have you been using this tool? @JamieWalker01 @nik-ahuja

**Comment by nik-ahuja** [comment]
Hi @vinayak-mehta yeah I have been using the tool. I personally think it's a useful feature. I have raised a [PR](https://github.com/vinayak-mehta/conrad/pull/160) just now.

**Comment by pranavkhekare** [comment]
Has this issue been solved ?
